File: abc139/abc139_c.cs
// https://atcoder.jp/contests/abc139/submissions/7264299

using System;
using System.Linq;

class Program
{
    static void Main()
    {
        var N = int.Parse(Console.ReadLine());
        var H = Console.ReadLine().Split(' ').Select(int.Parse).ToArray();

        var temp = 0;
        var result = 0;

        for (int i = 0; i < N - 1; i++)
        {
            if (H[i + 1] <= H[i])
            {
                temp++;
                if (temp > result)
                {
                    result = temp;
                }
            }
            else
            {
                temp = 0;
            }
        }
        Console.WriteLine(result);
    }
}

File: abc/abc139/abc139_c.fs
// https://atcoder.jp/contests/abc139/submissions/12017862

let N = stdin.ReadLine() |> int
let H = stdin.ReadLine().Split() |> Array.map float

let rec solver t r i =
    if i = N - 1 then r
    elif H.[i + 1] <= H.[i] then solver (t + 1) (max r (t + 1)) (i + 1)
    else solver 0 r (i + 1)

solver 0 0 0 |> stdout.WriteLine
